CONTRIBUTING.md

```diff
@@ -19,27 +13,33 @@ Note that some files are managed outside this repository and are committed autom
 
 `.github` directory is manage in [this repository](https://github.com/algolia/algoliasearch-client-common), any Pull Request there are welcome.
 
+## Opening an issue
+
+Each repository provide a template for issues. Please fill the client and language version and provide a clear description of the problem you're facing. Steps to reproduce or example code (repository, jsfiddle and such) are a big help.
+
 ## Submitting a pull request
 
-Keep your change as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
+Keep your changes as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
 
 Here are a few tips to increase the likelihood of being merged:
 
 - [ ] Write tests.
-- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) and follow (conventional commits)[https://conventionalcommits.org/].
```

Comment:
markdown link format is backwards here 🙂

Reply:
I'm personaly not a big fan of conventialcommits, but it seems like most engineers enjoy it here. I'm trying to organize my thoughts for the way to do changelogs in the API clients. I'll share them as soon as possible.